docs/queryworkers/running-queries.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 20
+title: Running Queries
+---
+
+After you [build a query](building-queries.md), you might want to run it. You might run a query to perform an action on your data (such as INSERT or DELETE), or you might test a query that you intend to save as a [Query Worker](query-workers.md).
+
+## Run a Query
+
+To run a query, perform the following steps.
+
+1. Write a [C8QL](../c8ql/) query.
+2. Enter any required [bind parameter](fundamentals.md#bind-parameters) values.
+3. Click **Run Query**.
+
+Macrometa runs the query and displays the [Query Result](#query-result) for that query. Each query you run displays another Query Result unless you navigate away from the page or clear the results.
```

Comment:
```suggestion
Macrometa runs the query and displays the [query result](#query-result) for that query. Each query you run displays another result unless you navigate away from the page or clear the results.
```

Reply:
"Query Result" is the name of the section on the screen.